signin.html

```diff
@@ -44,5 +48,6 @@ <h1>소셜 로그인</h1>
                 </div>
             </div>
         </div>
+          <script src="./script/signin.js"></script>
```

Comment:
일반 스크립트 대신에 module을 사용해보세요.

```suggestion
          <script type="module" src="./script/signin.js"></script>
```
<img width="916" alt="image" src="https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/assets/59413128/8615b162-5612-427e-8362-cd23e860187f">

https://ko.javascript.info/modules-intro#ref-992 에서 모듈에 대해 더 알아보시면 좋을 것 같아요 😄

Reply:
0c782ada4bb3f76d1855f6efd7dcbff9fb343b0e 일반 스크립트와 module의 차이점에 대해 더 자세하게 알게되었습니다! 모두 module로 수정하였습니다!